chapter09/mdw/Ex04.pl

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+$^I= ".out";
+
+my $first = 1;
```

Comment:
Yeah, I should have added a line after the print to reset $first if /\Z/, or something to that effect.